<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>433</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Twobody</string>
  </property>
  <property name="windowIcon">
   <iconset resource="twobody.qrc">
    <normaloff>:/images/twobody.png</normaloff>:/images/twobody.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="margin">
     <number>5</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QComboBox" name="leftComboBox"/>
          </item>
          <item>
           <widget class="PictureListView" name="leftListView">
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="syncButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="twobody.qrc">
              <normaloff>:/images/sync.png</normaloff>:/images/sync.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="addPicturesButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="twobody.qrc">
              <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QComboBox" name="rightComboBox"/>
          </item>
          <item>
           <widget class="PictureListView" name="rightListView">
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>709</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Add_pictures"/>
    <addaction name="actionTime_synchronize"/>
    <addaction name="separator"/>
    <addaction name="actionClear"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout_Twobody"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Quit">
   <property name="icon">
    <iconset resource="twobody.qrc">
     <normaloff>:/images/quit.png</normaloff>:/images/quit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionAbout_Twobody">
   <property name="icon">
    <iconset resource="twobody.qrc">
     <normaloff>:/images/twobody.png</normaloff>:/images/twobody.png</iconset>
   </property>
   <property name="text">
    <string>About Twobody</string>
   </property>
  </action>
  <action name="action_Add_pictures">
   <property name="icon">
    <iconset resource="twobody.qrc">
     <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Add pictures ...</string>
   </property>
   <property name="toolTip">
    <string>Add pictures</string>
   </property>
  </action>
  <action name="actionTime_synchronize">
   <property name="icon">
    <iconset resource="twobody.qrc">
     <normaloff>:/images/sync.png</normaloff>:/images/sync.png</iconset>
   </property>
   <property name="text">
    <string>Time synchronize</string>
   </property>
   <property name="toolTip">
    <string>Time synchronize</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="twobody.qrc">
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PictureListView</class>
   <extends>QListView</extends>
   <header>picturelistview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="twobody.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
